### PR TITLE
BMO deep dive presentation

### PR DESCRIPTION
**Pull request by hroyrh**
A presentation to cover the BMO in detail.

### CODE REVIEW

File: docs/presentations/bmo-deep-dive/bmo-deep-dive.html [hunk 1]
```diff
@@ -0,0 +1,407 @@
...
+$echo -n 'password' | base64
+cGFzc3dvcmQ=
+```
+
+
+
+### BareMetalHost Spec
+
+* bmc  : contains the connection information for the BMC (Baseboard Management Controller) on the host, its subfields are :
+	* address :
```
Comment (dtantsur) [inline docs/presentations/bmo-deep-dive/bmo-deep-dive.html:135]: Details could be useful here

File: docs/presentations/bmo-deep-dive/bmo-deep-dive.html [hunk 2]
```diff
@@ -0,0 +1,407 @@
...
+
+
+
+The operator supports several configuration options for controlling its interaction with Ironic.
+<br>
+<br>
+DEPLOY_RAMDISK_URL -- The URL for the ramdisk of the image containing the Ironic agent..
+<br>
+<br>
+DEPLOY_KERNEL_URL -- The URL for the kernel to go with the deploy ramdisk..
```
Comment (dtantsur) [inline docs/presentations/bmo-deep-dive/bmo-deep-dive.html:177]: I suggest you pick either "the Ironic agent" or "the deploy ramdisk"

File: docs/presentations/bmo-deep-dive/bmo-deep-dive.html [hunk 3]
```diff
@@ -0,0 +1,407 @@
...
+<br>
+<br>
+
+Implementation of this proposal is based on using annotation (similar to [Reboot API](https://github.com/metal3-io/metal3-docs/blob/master/design/baremetal-operator/reboot-interface.md)) to request inspection of a Ready BareMetalHost. Once the annotation is set on BareMetalHost, the baremetal operator will request hardware inspection of the host from Ironic.
+
+
+
+### Worker Config Drive
+
+Provisioning hosts requires two separate images. The first is the primary target image for the host, and contains the operating system and other software that the host will run. These are generally reusable across many hosts. The second image is the "config drive image", which contains configuration settings passed to the target image by writing an ISO to a separate partition accessible when the host boots. The config drive image is often, but not always, customized for each host. It is always often customized for the role a host plays (master, worker, etc.).
```
Comment (dtantsur) [inline docs/presentations/bmo-deep-dive/bmo-deep-dive.html:329]: "always often" - pick one :)

File: docs/presentations/bmo-deep-dive/bmo-deep-dive.html (hunk 1 above)
Comment (hroyrh) [reply docs/presentations/bmo-deep-dive/bmo-deep-dive.html]: @dtantsur I have made changes to the presentation, can you please check and let me know how it looks.

**Pull request closed without merging**